projects/gigya/src/auth/store/effects/index.ts

```diff
@@ -0,0 +1,6 @@
+
+import { GigyaUserTokenEffects } from './gigya-user-token.effect';
+
+export const effects: any[] = [GigyaUserTokenEffects];
```

Comment:
Could we drop this effects const and reference directly `GigyaUserTokenEffects`?